extensions/knative/model/src/main/java/io/fabric8/knative/serving/v1/ServiceStatus.java

```diff
@@ -0,0 +1,140 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.knative.serving.v1;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreType;
+import io.fabric8.knative.duck.v1.Addressable;
+import io.fabric8.knative.v1.Condition;
+import io.fabric8.kubernetes.api.model.*;
```

Comment:
nit, Could you please avoid wildcard imports? They become problematic whenever we are upgrading models and some apiVersion changes.

Reply:
sure, will push in a moment

Reply:
done